client/src/pages/AddPlace.js

```diff
@@ -0,0 +1,182 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import Input from '../components/general/Input';
+import TextArea from '../components/general/TextInput';
+import Button from '../components/general/Button';
+
+const Form = styled.form`
+  display: flex;
+  flex-flow: column;
```

Comment:
you don't use "wrap", so do you need `flex-flow`? Or is `flex-direction` sufficient?

Reply:
-direction would be sufficient, but Marwin said -flow is convention and I think it sounds nicer and is shorter ;)